server/sonar-server/src/main/java/org/sonar/server/db/migrations/PlatformDatabaseMigrationExecutorServiceImpl.java

```diff
@@ -32,6 +33,12 @@
   implements PlatformDatabaseMigrationExecutorService {
 
   public PlatformDatabaseMigrationExecutorServiceImpl() {
-    super(Executors.newSingleThreadExecutor());
+    super(
+      Executors.newSingleThreadExecutor(
+        new ThreadFactoryBuilder()
+          .setDaemon(false)
```

Comment:
the `ThreadFactory` implicitly used when calling `Executors.newSingleThreadExecutor` without arguments sets the `daemon` flag on created threads explicitly. Settings it here enforces the behavior is the same, only the name is changed.